### DISCUSSION TITLE
Jwts import 오류

### DISCUSSION THREAD

**Discussion by wansonam11** [fsoftwareengineer/todo-application#61] [질문과 답변]
![화면 캡처 2022-01-21 030346](https://user-images.githubusercontent.com/35058547/150396595-2b40bc7e-9d31-4de6-9ae8-25efb40ad658.png)

<p.251>
사진처럼 webtoken 관련은 아얘 import가 되질 않습니다.

- 현재 gradle 6.7버전이라 교재처럼 작성해도 안되고,  7.0 이상으로 올려서 공지대로 코드를 바꿔봐도 전혀 반응이 없습니다.
- 깃 코드에 있는 build.gradle / TokenProvider클래스 똑같이 복붙해봐도 여전히 webtoken부분만 import가 되질않습니다
- 이클립스 껏다켜봐도 반응이 없는데 왜이러는걸까요

이부분 skip하고 그냥 진행해볼까 하는데 
진도 나가는데에 혹시 지장이 있을까요,,?

**Comment by wansonam11** [comment]
혹시나 하는 마음에 build path경로에 jjwt.jar파일을 강제로 넣어줬더니 에러가 말끔히 사라졌습니다!

**Reply by fsoftwareengineer** [reply]
그래들이 jjwt를 인스톨하는지 확인 해 주시기 바랍니다.
